bench/thor/isochrone.cc

```diff
@@ -10,6 +10,7 @@
 #include "loki/worker.h"
 #include "thor/worker.h"
 
+#include "datatools/test.h"
```

Comment:
why would this be needed? the only thing it seems to use is `test::make_config` that one should not be in the  datatools target as it has nothing to do with datatools. actually let me take a closer look at what you moved into there...